src/controller.js

```diff
@@ -123,7 +123,9 @@ function drawLabels(ctx, item, rect) {
   if (!labelsOpts || !labelsOpts.display) {
     return;
   }
-  const font = toFont(labelsOpts.font);
+  const optColor = (rect.active ? labelsOpts.hoverColor : labelsOpts.color) || labelsOpts.color;
+  const optFont = (rect.active ? labelsOpts.hoverFont : labelsOpts.font) || labelsOpts.font;
```

Comment:
Doesn't the core option resolution handle the active case already? It should prefer any `hover*` options when the element is active by default (and set it without the hover prefix).

Reply:
I have tried without that check and it doesn't work. Should be weird to me how this can be done "automatically" because the font (for instance) change the position to draw the label and this logic is implemented in the controller.

Reply:
Tested again with this config:

```javascript
labels: {
	display: true,
	formatter: 'primo',
	font: {
		size: 12,
		//weight: 'bold'
	},
	hoverFont: {
		size: 24,
		weight: 'bold'
	},
	align(ctx) {
		return 'right';
	},
	position: 'bottom',
	color(ctx) {
		return 'yellow';
	}
}
```

Code:

```javascript
  const optColor = (rect.active ? labelsOpts.hoverColor : labelsOpts.color) || labelsOpts.color;
  //const optFont = (rect.active ? labelsOpts.hoverFont : labelsOpts.font) || labelsOpts.font;
  const optFont = labelsOpts.font;
```

and I confirm it does not work, the font is not changed when hovered.

Reply:
ok, thanks for testing!